tests/test_cff.py

```diff
@@ -73,12 +73,15 @@ def test_broken_cff(cff_file):
 def test_parse_doi():
     cff_file = b"""
     cff-version: 1.2.0
-    title: gimie
-    doi: 10.5281/zenodo.1234567
+    message: If you use this software, please cite it using these metadata.
+    title: 'napari: a multi-dimensional image viewer for Python'
+    identifiers:
+    - type: doi
+      value: 10.5281/zenodo.3555620
```

Comment:
**suggestion:** probably better to have a test example with multiple DOIs. This way we will catch it if one day we break our chosen behavior (return all, or first one).